### PR TITLE
Makes execution sword a bit less spammy.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
tweak: The executioners sword will not play a song if another sword caused music to start playing. Also the sword has a short cooldown between execution attempts. 
tweak: The executioners sword will now stop playing music if the execution is interrupted. 
fix: Also fixed that annoying click in nasheed2.ogg
/:cl:

[why]: 10 nasheeds at once was funny once.

### CODE REVIEW

File: hippiestation/code/_globalvars/misc.dm [hunk 1]
```diff
@@ -0,0 +1 @@
+GLOBAL_VAR_INIT(nasheed_playing, FALSE)
```
Comment (ghost) [inline hippiestation/code/_globalvars/misc.dm:1]: You don't need a file dedicated to this, you can shove this at the top of the Executionerr's sword file

**Review by ghost — DISMISSED** [review]

File: hippiestation/code/_globalvars/misc.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/_globalvars/misc.dm:1]: This is fine

**Review by ghost — DISMISSED** [review]

**Comment by KayeArray** [comment]
+1

File: hippiestation/code/game/objects/items/execution_sword.dm [hunk 2]
```diff
@@ -68,21 +72,31 @@ obj/item/melee/execution_sword/attack_self(mob/living/user)
...
-			playing_nasheed = TRUE
+			GLOB.nasheed_playing = TRUE
 			addtimer(CALLBACK(src, .proc/nasheed_end), EXECUTE_INFIDEL)
 		if(do_after(user,EXECUTE_INFIDEL, target = target))
 			log_admin("[key_name(user)] executed [key_name(target)] with [src]")
 			message_admins("[key_name(user)] executed [key_name(target)] with [src]")
 			infidel_head.dismember()
 			priority_announce("[user] has executed [target] in the name of [execution_faction]!","Message from [execution_faction]!", 'sound/misc/notice1.ogg')
 			executing = FALSE
+			can_execute = FALSE
```
Comment (ghost) [inline hippiestation/code/game/objects/items/execution_sword.dm]: why is this code in both if and else instead at the end if it should always run at this point?

**Review by tortellinitony — DISMISSED** [review]
- Make it only play locally maybe?
- Make the sound play in its own CHANNEL
- Clear that channel when the sound is supposed to stop playing

**Comment by tortellinitony** [comment]
Also please don't make it spam a Centcom message when someone starts executing...make it only do that when you finish. Please for the love of God oh please oh please almighty make the song only play locally

**Comment by GuyonBroadway** [comment]
The whole point of the sword was to announce to the station that you are going to kill someone and where its happening.

I can try to make the sound play on its own channel tho and mute it when its interrupted.

**Comment by tortellinitony** [comment]
I don't see why it has to be that way, though. It's really annoying when someone just spams it and stops and it announces and starts the music every time. It should only play the music locally, and if you really want it to announce where someone is, do that when they finish the execution

**Comment by GuyonBroadway** [comment]
Gave nasheed its own channel that's silenced upon an execution being interrupted.

**Comment by dreamcrab** [comment]
yea this shit is just an awful idea all around and people spam the initial message which is really annoying

fix that shit and you have my blessings

amen

**Comment by GuyonBroadway** [comment]
I did by give the sword a cooldown between attempts. I can extend it if need be.

File: hippiestation/code/game/objects/items/execution_sword.dm (hunk 2 above)
Comment (GuyonBroadway) [reply hippiestation/code/game/objects/items/execution_sword.dm]: whups

**Review by ghost — APPROVED** [review]
ok.

**Pull request merged**